.github/workflows/label-feature-request.yml

```
@@ -0,0 +1,26 @@
+name: 🤖 Issue labeler - Feature Request
+on:
+  issues:
+    types:
+      - labeled
+jobs:
+  add-comment:
+    if: github.event.label.name == "triaged: feature request"
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+      - name: Add comment
+        uses: peter-evans/create-or-update-comment@5adcb0bb0f9fb3f95ef05400558bdb3f329ee808
+        with:
+          issue-number: ${{ github.event.issue.number }}
+          comment: |
+            Thank you for taking the time to file this issue!             
+            
+            We have triaged your feature request and added it to our backlog.
+            
+            We don't have a timeline for when this feature will be implemented, please let us know if you would be interested in sending a PR.
+            
+            We will need to do planning and designing before we accept a PR, so please be patient with us!
+            
+            See the [contributing guidelines](https://github.com/vercel/vercel/blob/main/.github/CONTRIBUTING.md) for more info.
```

```suggestion
            Thank you for taking the time to created this request!             
            
            We added it to our backlog but need to discuss design/architecture before we can accept a PR. 
            
            Please let us know if you would be interested in sending a PR once we approve the design.
```

I'm assuming `triaged: feature request` is not the same as `accepted - send a PR`, right?